### ISSUE TITLE
Perform ticket migration

### ISSUE THREAD

**Issue by danuker** [chevah/trac-to-github#16]
- [ ] Disable chevah robi-net automation
- [ ] Dry-run, check assigned issue numbers
- [ ] Condition tickets to migrate the PR -> sftpplus.com ones first. Migrate single ticket, see if all is all right. Then also allow the next ticket, see if the incremental migration works. Then migrate all.
- [ ] As a safety precaution, change the config to invalid repos for ones not intended to migrate.

**Comment by adiroiban** [comment]
Before the migration we will make sure that both Trac and Github are not updated

**Comment by danuker** [comment]
No, this is a "story" ticket keeping track of progress (so far, none).

**Comment by adiroiban** [comment]
Ok. Np.

**Comment by adiroiban** [comment]
The following Trac ticket metadata should be converted into labels

* status - closed | fixed | new | duplicate | re-open ... etc
* type - defect | enhancement | task  | release blocker
* priority
* component

Milestone should be assigned to GitHub Milestones

For all the other "free text" metadata we will only have the as machine readable text:

* reporter
* author
* keywords
* cc
* branch


But we should also include status, type, priority...etc as machine readable.
